parsing/dialects/bigquery/src/test/java/org/apache/calcite/test/BigQueryParserTest.java

```diff
@@ -27,4 +29,31 @@
   @Override protected SqlParserImplFactory parserImplFactory() {
     return BigQueryParserImpl.FACTORY;
   }
+
+  @Test public void testExceptSingle() {
+    final String sql = "SELECT * EXCEPT(a) FROM foo";
+    final String expected = "SELECT * EXCEPT (`A`)\n"
+     + "FROM `FOO`";
```

Comment:
I think the indentation level of the + is causing tests to fail (tests say indentation level should be 8). Same for other tests below.